README.md

```diff
@@ -1,6 +1,10 @@
-# clingo-sys
+# clingo-sys)
```

Comment:
Was just looking at what you had to do to get things running on windows. Maybe you want to remove the `)` here.

Reply:
yep